code/src/App.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { combineReducers, configureStore } from "@reduxjs/toolkit";
+import { game } from "./reducers/game";
+import { ui } from "./reducers/ui";
+import { GameContainer } from "components/GameContainer";
+
+const reducer = combineReducers({
```

Comment:
From suggestions perspective , it would be cool to add a localstorage , however it doesnt affect an overall gaming experience cause as a user I didnt actually need to overload the page in the process . Might be it can be useful if users connection is unstable or somewhat. Anyway might be not that vital but it is a matter of few code lines , so this is a possible area for further development